### DISCUSSION TITLE
Centrality measures for pre-partions in network

### DISCUSSION THREAD

**Discussion by liking4viking** [gephi/gephi#2763] [Q&A]
Dear all, 

I have pre-partitioned my network by gender and would like to analyse the centrality measures not only for the entire network but also for men and women to see if there are any differences between these groups.

However, I am having a few issues trying to get these measures and I am clearly clicking on the wrong statistics or partitions. 

Any help would be appreciated and please let me know if you need more information. 

Thank you in advance.

**Comment by mbastian** [comment]
Hi @liking4viking, once you filter your graph based on that partition (Filters > Attributes > Partition > ... > Filter) the statistics you run are only using that "filtered graph" in account. What you should probably do then is to rename the output column manually. You can do that in the Data Laboratory. Rename it "Men Centrality" for instance and then re-run the centrality algorithm for the other partition. Renaming the column is helpful otherwise you have both results in one column, which can make it confusing and harder to compare. Hope this helps,
